### PR TITLE
Adding in api.txt files to ktx and app distribution directories 

### PR DESCRIPTION
**Pull request by mannyjimenez0810**
All sdks have an api.txt files that include methods that users will be interacting with both on java and in kotlin.

### CODE REVIEW

File: firebase-app-distribution/api.txt [hunk 1]
```diff
@@ -0,0 +1,15 @@
+// Signature format: 2.0
+package com.google.firebase.app.distribution {
+  public class FirebaseAppDistribution{
+    method @NonNull public static com.google.firebase.app.distribution.FirebaseAppDistribution getInstance();
+    method @NonNull public static com.google.firebase.app.distribution.FirebaseAppDistribution getInstance(@NonNull com.google.firebase.FirebaseApp);
```
Comment (pranavrajgopal) [inline firebase-app-distribution/api.txt:5]: line 6 and line 7 need @NonNull annotations

File: firebase-app-distribution/ktx/api.txt [hunk 2]
```diff
@@ -0,0 +1,10 @@
+// Signature format: 2.0
+package com.google.firebase.app.distribution.ktx {
+
+  public final class FirebaseAppDistributionKt {
+    ctor public FirebaseAppDistributionKt();
```
Comment (pranavrajgopal) [inline firebase-app-distribution/ktx/api.txt:5]: nit: formatting on the file seems to be off. get rid of the blank space at the start of line 6 and line 7

File: firebase-app-distribution/ktx/api.txt [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+       method @NonNull public static com.google.firebase.app.distribution.FirebaseAppDistribution getAppDistribution(@NonNull com.google.firebase.ktx.Firebase);
+       method @NonNull public static com.google.firebase.app.distribution.FirebaseAppDistribution appDistribution(@NonNull com.google.firebase.ktx.Firebase, @NonNull com.google.firebase.FirebaseApp app);thod @NonNull public static com.google.firebase.crashlytics.FirebaseCrashlytics getCrashlytics(@NonNull com.google.firebase.ktx.Firebase);
```
Comment (pranavrajgopal) [inline firebase-app-distribution/ktx/api.txt:7]: not sure what thod is :) Also we should remove the crashlytics reference

**Review by pranavrajgopal — APPROVED** [review]

**Review by rachaprince — APPROVED** [review]

File: firebase-app-distribution/api.txt [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+    method @NonNull public com.google.firebase.app.distribution.UpdateTask updateIfNewReleaseAvailable();
+    method @NonNull public com.google.android.gms.tasks.Task<java.lang.Void> signInTester();
+    method @NonNull public com.google.android.gms.tasks.Task<com.google.firebase.app.distribution.AppDistributionRelease> checkForNewRelease();
+    method @NonNull public com.google.firebase.app.distribution.UpdateTask updateApp();
+    method public java.lang.Boolean istesterSignedIn()
```
Comment (ehsannas) [inline firebase-app-distribution/api.txt:10]: Missing `;` at the end of the line here.

**Comment by ehsannas** [comment]
I see CI bots failing on my PR for Firestore because the bots run `apiInformation` for all firebase components (e.g. `./gradlew :firebase-app-distribution:apiInformation`)
https://storage.googleapis.com/android-ci/pr-logs/pull/firebase_firebase-android-sdk/3185/api-information/1467983777874579459/build-log.txt

**Comment by ehsannas** [comment]
AFAIK the api.txt files should be auto-generated by running `./gradlew :firebase-app-distribution:generateApiTxtFile`

**Comment by ehsannas** [comment]
I have posted https://github.com/firebase/firebase-android-sdk/pull/3213 to fix this, but I'm not familiar with app distribution APIs. Please review.

**Pull request merged**